ste/downloader-azureFiles_windows.go

```diff
@@ -103,9 +103,9 @@ func (*azureFilesDownloader) PutSDDL(sip ISMBPropertyBearingSourceInfoProvider,
 		nil,
 	)
 
-	if err != nil && strings.HasPrefix(sddlString, "O:SYG:SY") {
-		// TODO: awaiting replies re where this SSDL comes from
-		return errorCantSetLocalSystemSddl
+	if err != nil {
+		return fmt.Errorf("permissions could not be restored. It may help to run from a elevated command prompt, and set the '%s' flag. Error message was: %w",
+			common.BackupModeFlagName, err)
 	}
```

Comment:
We decided to persist these anyway?

Reply:
Yes, subject to possible review in a future PR, the plan for now is that when Files gives us a default ACL, we will use it.  (Otherwise, if some have default ACLs and some have user-specifed ACLS, we end up setting no ACL on the defaulted ones - which is probably counter-intuitive to the user)